anomaly_detector/storage/local_storage.py

```diff
@@ -81,10 +81,13 @@ class LSConfiguration(Configuration):
     """Configuration for local storage."""
 
     # Path to a file containing input data. If the value is '-' the input is expected on STDIN
-    LS_INPUT_PATH = Configuration.LOCAL_DATA_FILE #"validation_data/verification_data.json"
+    LS_INPUT_PATH = ""
     # Path to a file where the results will be stored. Results will be printed to STDOUT if empty
-    LS_OUTPUT_PATH = Configuration.LOCAL_RESULTS_FILE
+    LS_OUTPUT_PATH = ""
 
-    def __init__(self):
+    def __init__(self,config):
         """Initialize configuration."""
-        self.load()
+        self.load_from_env()
+        self.LS_INPUT_PATH=config.LOCAL_DATA_FILE
```

Comment:
If you rename this config and the other to `LS_INPUT_PATH` you don't have to make this assignment. Just keep the ENV variables and private variable names in sync

Reply:
@durandom Good point. Yes. Going to try this

Reply:
I've removed this code. We inject configs in a better way.

Reply:
Done